ega-data-api-res/src/main/java/eu/elixir/ega/ebi/reencryptionmvc/cache2k/My2KCachePageFactory.java

```diff
@@ -118,6 +115,10 @@ private String getServiceURL(String service) {
                 }
             } while (!pageSuccess && pageCnt++ < 3);
 
+            if(!pageSuccess) {
```

Comment:
I saw that you have run "Format Code" on the other classes. That's great!
Maybe you should run it also on this class?
And then this would become
```
if (!pageSuccess) {
    throw new ServerErrorException("FIRE error can't read data, file id " + id + ", page " + cachePage);
```